### PR TITLE
[batch] rename routes

### PR DESCRIPTION
**Pull request by jigold**
@cseed I didn't know what to do with healthcheck. I think we either need two routes or keep it at `/healthcheck`

### CODE REVIEW

File: batch/test/test_batch.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def test_ui_batches(self):
         with open(os.environ['HAIL_TOKEN_FILE']) as f:
             token = f.read()
         # just check successful response
-        r = requests.get(f'{os.environ.get("BATCH_URL")}/ui/batches',
+        r = requests.get('/batches',
```
Comment (cseed) [inline batch/test/test_batch.py]: I think you still need the batch url in there.

**Comment by jigold** [comment]
I fixed the two CI callback URLs. It looks like the GitHub one isn't even working. Once this PR goes in, the web hook for ci needs to be modified.

File: batch/batch/server/server.py [hunk 2]
```diff
@@ -911,13 +911,6 @@ def is_successful(self):
     return {'batch_id': batch_id, 'job_id': job_id, 'job_log': job_log}
 
 
-@routes.get('/')
```
Comment (danking) [inline batch/batch/server/server.py]: let's keep this!

**Comment by danking** [comment]
@cseed What are you thinking of in pipeline? Both CI and pipeline should use the batch client and thus be abstracted from changes in URLs.

**Pull request merged**